.github/workflows/pytest-gpu.yml

```diff
@@ -46,31 +46,34 @@ jobs:
           pytest-gpu-acc-nvidia,
           pytest-gpu-omp-amd
         ]
+        test_examples: ["examples/seismic/tti/tti_example.py examples/seismic/acoustic/acoustic_example.py examples/seismic/viscoacoustic/viscoacoustic_example.py examples/seismic/viscoelastic/viscoelastic_example.py examples/seismic/elastic/elastic_example.py"]
+
         include:
         - name: pytest-gpu-omp-nvidia
           test_files: "tests/test_adjoint.py tests/test_gpu_common.py tests/test_gpu_openmp.py"
           base: "devitocodes/bases:nvidia-clang"
           tags: ["self-hosted", "nvidiagpu"]
           test_drive_cmd: "nvidia-smi"
           flags: '--gpus all --rm --name testrun-clang-nvidia'
+          mpi_flags: ''
 
         - name: pytest-gpu-acc-nvidia
           test_files: "tests/test_adjoint.py tests/test_gpu_common.py tests/test_gpu_openacc.py"
           base: "devitocodes/bases:nvidia-nvc"
           tags: ["self-hosted", "nvidiagpu"]
           test_drive_cmd: "nvidia-smi"
           flags: '--gpus all --rm --name testrun-nvc'
+          mpi_flags: ''
 
         - name: pytest-gpu-omp-amd
-          # We only test the OpenMP generation due to issue with the offloading discussed in
-          # https://github.com/ROCm-Developer-Tools/aomp/issues/397
-          test_files: "tests/test_gpu_openmp.py"
+          test_files: "tests/test_adjoint.py tests/test_gpu_common.py tests/test_gpu_openmp.py"
           tags: ["self-hosted", "amdgpu"]
           base: "devitocodes/bases:amd"
           test_drive_cmd: "rocm-smi"
           # Attach the AMD GPU devices `/dev` and add user to video and render (109 on wampa) group
           # Options from https://rocmdocs.amd.com/en/latest/ROCm_Virtualization_Containers/ROCm-Virtualization-&-Containers.html
           flags: "--network=host --device=/dev/kfd --device=/dev/dri --ipc=host --group-add video --group-add 109 --cap-add=SYS_PTRACE --security-opt seccomp=unconfined --rm --name testrun-amd"
+          mpi_flags: "--mca btl '^openib' -x UCX_TLS=sm,self,rocm_copy,rocm_ipc  --mca pml ucx"
```

Comment:
Can we make them part of the dockerfile somehow?

Reply:
We could be this makes it slightly easier for ci

Reply:
but OK why do these options only matter for CI and not for the end users?

Reply:
No it does too and should be moved to the docker at some point, but I would need to re-build the docker separately , test it, then restart that PR so it's not very practical that's all

Reply:
As mentionned in slack these should go in the `mpiexec.conf`

Reply:
fixed